services/smart_contract_service.proto

```diff
@@ -89,4 +89,9 @@ service SmartContractService {
      * Undeletes a contract if the submitting account has network admin privileges
      */
     rpc systemUndelete (Transaction) returns (TransactionResponse);
+
+    /**
+     * Ethereum transaction
+     */
+    rpc contractCallEthereum (Transaction) returns (TransactionResponse);
```

Comment:
Are all ethereum transactions invoked on an EVM? I thought some could be native transactions like crypto transfer? Perhaps we call this `ethereumCall` or something without `contract` if so?

Reply:
All Ethereum Transactions are executed in context of the EVM, even crypto transfers.  May not be a contract though, so I'll go to `callEthereum`